apk-generator/v2/scripts/fdroid_build_apk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+./gradlew assembleRelease --info
```

Comment:
@arp95 could you please do what @Shailesh351 has said ? build only that specific flavour.